scripts/_terraform.sh

```diff
@@ -456,6 +456,8 @@ function _terraform_cleanup() {
                 echo "Environment $env is a test environment.  It will be destroyed... "
                 echo
                 uhd terraform destroy:layer 20-app $env
+            elif [[ $env == etl-ci-* ]]; then
```

Comment:
This isn't strictly needed but it brings a little more clarity to the envs we are skipping and why.
In essence, I've set this up so that the main infra is responsible for cleaning up its own CI envs.
Whereas the ETL infra is responsible for cleaning up the ephemeral CI test environments that it creates in its own CI pipeline.